src/Audio/Cue.cs

```diff
@@ -774,6 +783,14 @@ internal bool INTERNAL_update()
 								long elapsedFromStop = INTERNAL_timer.ElapsedMilliseconds - INTERNAL_fadeStart;
 								variableValue = elapsedFromStop;
 							}
+							else
+							{
+								variableValue = 0.0f;
+							}
+						}
+						else if (curRPC.Variable.Equals("NumCueInstances"))
```

Comment:
Makes sense, especially considering the special case I wrote wasn't even correct... >_<

Go ahead and remove this `else` block and I'll merge this all in.